lib/bot.js

```diff
@@ -50,6 +55,16 @@ Bot.prototype.attachListeners = function() {
   this.client.addListener('error', function(error) {
     logger.error('Received error event from IRC', error);
   });
+
+  this.client.addListener('invite', function(channel, from, message) {
+    logger.debug('Received invite', channel, from, message);
+    if (this.opt.channels.indexOf(channel) > -1) {
```

Comment:
Yep, forgot that the `channelMapping` keys where Slack-channels and not IRC ones.
You can use `this.channels`, from [here](https://github.com/ekmartin/slack-irc/blob/8003e0b8841b367ed125cd8dd35ad6f7af50da19/lib/bot.js#L27), which will be an array of all the IRC channels from the channel mapping, instead of `this.opt.channels`.